### PR TITLE
Refactored initial commit

### PR DESCRIPTION
**Pull request by Iamyuvraj143**
Order Controller Refactor code

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -40,4 +40,10 @@ def redirect_handler(redirect_path, message)
     flash[:notice] = message
   end
 
+  def null_value_check(value , path, message)
+    unless value.present?
+      redirect_handler(path, message)
+    end
+  end
```
Comment (kalpana19sharma) [inline app/controllers/application_controller.rb]: Instead of this you can raise an exception and rescue that exception here.

**Review by kalpana19sharma — CHANGES_REQUESTED** [review]
Fix the styleguide offenses in all files.

**Comment by Iamyuvraj143** [comment]
> Fix the styleguide offenses in all files.

Done !!

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (Iamyuvraj143) [reply app/controllers/application_controller.rb]: Done

**Pull request merged**